README.md

```diff
@@ -4,247 +4,148 @@ This backend is currently in development and is not advised for Production workf
-
-For example, `settings.py` in the project template specifies:
+Navigate to your `example/settings.py` file and find the variable named `DATABASES` Replace the `DATABASES` setting with this:
 
 ```python
-MIGRATION_MODULES = {
-    "admin": "mongo_migrations.admin",
-    "auth": "mongo_migrations.auth",
-    "contenttypes": "mongo_migrations.contenttypes",
+DATABASES = {
+   "default": django_mongodb_backend.parse_uri("<CONNECTION_STRING_URI>"),
 }
 ```
 
-The project template includes these migrations, but you can generate them if
-you're setting things up manually or if you need to create migrations for
-third-party apps. For example:
-
-```console
-$ python manage.py makemigrations admin auth contenttypes
-Migrations for 'admin':
-  mongo_migrations/admin/0001_initial.py
-    - Create model LogEntry
-...
-```
-
-### Creating Django applications
-
-Whenever you run `python manage.py startapp`, you must remove the line:
-
-`default_auto_field = 'django.db.models.BigAutoField'`
-
-from the new application's `apps.py` file (or change it to reference
- `"django_mongodb_backend.fields.ObjectIdAutoField"`).
-
-Alternatively, you can use the following `startapp` template which includes
-this change:
+Where `<CONNECTION_STRING_URI>` is your connection string from the previous step.
 
+### Start the Server
+To verify that you installed Django MongoDB Backend and correctly configured your project, run the following command from your project root:
 ```bash
-$ python manage.py startapp myapp --template https://github.com/mongodb-labs/django-mongodb-app/archive/refs/heads/5.0.x.zip
+python manage.py runserver
 ```
-(where "5.0" matches the version of Django that you're using.)
+Then, visit http://127.0.0.1:8000/. This page displays a "Congratulations!" message and an image of a rocket.
 
-### Configuring the `DATABASES` setting
 
-After you've set up a project, configure Django's `DATABASES` setting similar
-to this:
+## Capabilities of Django Backend for MongoDB
 
-```python
-DATABASES = {
-    "default": {
-        "ENGINE": "django_mongodb_backend",
-        "HOST": "mongodb+srv://cluster0.example.mongodb.net",
-        "NAME": "my_database",
-        "USER": "my_user",
-        "PASSWORD": "my_password",
-        "PORT": 27017,
-        "OPTIONS": {
-            # Example:
-            "retryWrites": "true",
-            "w": "majority",
-            "tls": "false",
-        },
-    },
-}
-```
+- **Database Connectivity**  
+    
+  - Directly tune MongoDB connection settings within your Django configuration\!  
+  - Work against a persisted cloud instance of MongoDB for free\!
 
-For a localhost configuration, you can omit `HOST` or specify
-`"HOST": "localhost"`.
 
-`HOST` only needs a scheme prefix for SRV connections (`mongodb+srv://`). A
-`mongodb://` prefix is never required.
+- **Model MongoDB Documents Through Django’s ORM**  
+    
+  - Translate Django model instances to MongoDB documents.  
```

Comment:
Translate is a strange work. How about "Store Django model instances as MongoDB documents."?